components/ActivityIndicator/src/MDCActivityIndicator.m

```diff
@@ -404,13 +441,13 @@ - (void)updateStrokePath {
 }
 
 - (void)updateStrokeColor {
```

Comment:
Based on _cycleColors never being empty, simplify to self.strokeColor = self.cycleColors[currentColorCount];